zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -449,7 +449,15 @@ ascan.options.strength.high.label = (More attacks)
 ascan.options.strength.insane   = Insane
 ascan.options.strength.insane.label = (MUCH more attacks)
 ascan.options.strengthTo.label		= Strength To
-ascan.options.thresholdTo.label		= Threshold To 
+ascan.options.thresholdTo.label		= Threshold To
+
+scan.filter.filtercriteria.include = Request should include one of the following {0} values :- {1} 
+scan.filter.filtercriteria.exclude = Request should not include the following {0} value :- {1}  
+scan.filter.filtercriteria.includeall = Request should include all of the following {0} values :- {1}
+scan.filter.filterType.Tag = Tag; 
```

Comment:
Are the semicolons really needed?

Reply:
ahh .. correct. Java has made my habit for adding semicolons after each line.